test/npcomp-run-mlir/broadcast.mlir

```diff
@@ -12,6 +12,6 @@
 // CHECK: output #0: dense<[
 // CHECK-SAME: [4.000000e+00, 5.000000e+00], [1.300000e+01, 1.400000e+01]]> : tensor<2x2xf32>
 func @broadcast(%arg0: tensor<?x?xf32>, %arg1: tensor<?x?xf32>) -> tensor<?x?xf32> {
-  %0 = "tcf.add"(%arg0, %arg1) : (tensor<?x?xf32>, tensor<?x?xf32>) -> tensor<?x?xf32>
+  %0 = tcf.add %arg0, %arg1 : (tensor<?x?xf32>, tensor<?x?xf32>) -> tensor<?x?xf32>
   return %0 : tensor<?x?xf32>
 }
```

Comment:
newline

Reply:
It's actually very hard to avoid this in my editor apparently (VSCode). It seems pretty innocuous so I'm going to leave it (especially since it was already there. Maybe we can have a formatter script that does this like we do for clang-format?